### DISCUSSION TITLE
Welcome to Osintgram Discussions!

### DISCUSSION THREAD

**Discussion by Mohamad-Mortada** [thedev132/Osintgram#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xxxbats** [comment]
is there a youtube video on the installation process and everything else? im new to all this so it would be much appreciated if i could get some help.

**Reply by thedarkknight9088** [reply]
have you managed to install it? i havent and really need some help
